Sources/TootSDK/Models/MediaAttachment.swift

```diff
@@ -56,6 +60,8 @@ public struct AttachmentMeta: Codable, Hashable, Sendable {
     public var original: AttachmentMetaInfo?
     public var small: AttachmentMetaInfo?
     public var focus: AttachmentMetaFocus?
+    /// For ``AttachmentType/audio`` attachments, contains colors extracted from the thumbnail.
```

Comment:
Is `/audio` correct here?

Reply:
Yes, at least with mastodon the `colors` are only present for audio and match the artwork associated with the audio file (like album art). I just used the terminology from the mastodon PR that added the feature, is there a different way of saying it that would be clearer?

Reply:
Oh I see! Something like this?

```suggestion
    /// For ``AttachmentType/audio`` attachments, contains colors matching the artwork (thumbnail) associated with the audio file.
```